ignite/pkg/xos/mv.go

```diff
@@ -0,0 +1,33 @@
+package xos
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+// Rename copy sourcePath to destPath and then delete sourcePath.
+// Unlike os.Rename, it doesn't fail when the oldpath and newpath are in
+// different partitions (error: invalid cross-device link).
+func Rename(oldpath, newpath string) error {
```

Comment:
Can we add a test for this?

Reply:
sure [fc5e5e7](https://github.com/ignite/cli/pull/2749/commits/fc5e5e77c6d839ed9799220dff2a696729cd0e2a)